src/views/InvolvementProfile/components/Membership/components/NonMemberButtons/index.jsx

```diff
@@ -74,70 +83,82 @@ const NonMemberButtons = ({
   };
 
   return (
+    //the Card tag and div section is used for the confetti-it is the same tag used for the happy birthday banner, just moved here
     <>
-      <CardActions>
-        {isGuest ? (
-          <Button variant="contained" color="primary" onClick={onUnsubscribe}>
-            Unsubscribe
-          </Button>
-        ) : (
+      <Card>
+        <div style={{ position: 'fixed', zIndex: 999999, top: -60, left: '50vw' }}>
+          <GordonConfetti active={confetti} colorOption="Gordon" />
```

Comment:
You were trying to get this to be only the GordonBlue color palette before. Do you still want it to be just blue and cyan? If so, you can make it `colorOption="GordonBlue"` instead of `colorOption="Gordon"`

Reply:
That makes sense. I'm good with leaving it as 'Gordon' for the multicolor confetti, it would fit better for the various clubs that aren't explicitly Gordon-specific like you said.